.changeset/all-loops-deny.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': minor
+---
+
+Updates the `App.match` and `NodeApp.match` method to accept a second, optional parameter.
```

Comment:
This could do with being clearer about how this is something that only affects people building adapters. I'm not really sure it needs to be a minor tbh

Reply:
I'm on the fence. I don't mind releasing it as a patch, because it's a very tiny feature and it affects a very low number of people.